app/views/convictions/index.html.erb

```diff
@@ -89,23 +89,23 @@
             <%= @resource.metaData.revoked_reason %>
           </td>
         </tr>
-        <% if @resource.conviction_sign_offs.first.confirmed_by.present? %>
+        <% if @resource.sign_off.confirmed_by.present? %>
         <tr>
           <td>
             <%= t(".conviction_sign_off.labels.confirmed_by") %>
           </td>
           <td>
-            <%= @resource.conviction_sign_offs.first.confirmed_by %>
+            <%= @resource.sign_off.confirmed_by %>
           </td>
         </tr>
         <% end %>
-        <% if @resource.conviction_sign_offs.first.confirmed_at.present? %>
+        <% if @resource.sign_off.confirmed_at.present? %>
         <tr>
           <td>
             <%= t(".conviction_sign_off.labels.confirmed_at") %>
           </td>
           <td>
-            <%= @resource.conviction_sign_offs.first.confirmed_at.in_time_zone("London").strftime("%A %e %B %Y at %l:%M%P") %>
+            <%= @resource.sign_off.confirmed_at.in_time_zone("London").strftime("%A %e %B %Y at %l:%M%P") %>
```

Comment:
Could we move this to `config/initializers/time_formats.rb`? Then use

```suggestion
            <%= @resource.sign_off.confirmed_at.in_time_zone("London").to_formatted_s(:weekday_day_month_year_at_time) %>
```
I find it makes it a little clearer what the format will be, rather than having to spin IRB or google it

N.B. Feel free to pick a better name!

Reply:
Good shout. Fixed: https://github.com/DEFRA/waste-carriers-back-office/pull/489/commits/d394a3407426684ace7c4b384fb87c1778dc02d6